cli/smartcontract/smart_contract.go

```diff
@@ -449,9 +453,15 @@ func initSmartContract(ctx *cli.Context) error {
 	if err := ioutil.WriteFile(filepath.Join(basePath, "neo-go.yml"), b, 0644); err != nil {
 		return cli.NewExitError(err, 1)
 	}
+
+	ver := ModVersion
+	if ver == "" || strings.HasSuffix(ver, "_unsupported") {
+		ver = "latest"
```

Comment:
Are you sure that it's appropriate to use `latest` interops for unsupported versions? If we build it for windows, then we might still want to use the appropriate interop package for windows because `latest` might be incompatible with what we're building right now.

Reply:
Fixed. So if `ver` is empty, use `latest` (any other reasonable suggestions?).